src/utils/organization.tsx

```diff
@@ -11,7 +11,15 @@ export const getFacilitiesByType = (organization: Organization, type: FacilityTy
 };
 
 export const getAllSeedBanks = (organization: Organization): Facility[] => {
-  return getFacilitiesByType(organization, 'Seed Bank');
+  return getFacilitiesByType(organization, 'Seed Bank').sort((a, b) => {
```

Comment:
same.. might as well add a string utils hook which automatically fetches user locale and does a localecompare
useStringCompare

Reply:
I was trying to do this, but a hook can't be called here

Reply:
thanks for trying.. ok looks like we will have to pass the user locale into all these functions then, sigh

Reply:
another option is to leave the sorting up to the rendering component, or however you think works best

Reply:
I'll pass the locale into these functions